settings.example.json

```diff
@@ -1,166 +1,80 @@
-			"req_per_sec": 10,
-			"burst": 10
-		},
-		"ipfs_timeout_secs": 300,
-		"perform_archival": false,
-		"perform_ipfs_unpin": true,
-		"prune_redis_zsets": true,
-		"backup_redis_zsets_to_file": false,
-		"oldest_project_index": "7d",
-		"pruning_height_behind_oldest_index": 100,
-		"summary_projects_prune_height_behind_head": 1000,
-		"web3_storage": {
-			"timeout_secs": 600,
-			"rate_limit": {
-				"req_per_sec": 1,
-				"burst": 1
-			},
-			"upload_chunk_size_mb": 50
-		}
-	},
-	"use_consensus": true,
-	"consensus_config": {
-		"service_url": "http://offchain-consensus:9030",
-		"rate_limit": {
-			"req_per_sec": 5,
-			"burst": 5
-		},
-		"timeout_secs": 30,
-		"max_idle_conns": 5,
-		"idle_conn_timeout": 0,
-		"finalization_wait_time_secs": 300,
-		"polling_interval_secs": 10
-	},
-	"token_aggregator": {
-		"port": 8000,
-		"run_interval_secs": 60,
-		"ap_host": "localhost"
-	}
+  "instance_id": "generated-uuid",
+  "pooler_namespace": "relevant-namespace",
+  "anchor_chain_rpc_url": "https://rpc-url",
+  "local_cache_path": "/home/ubuntu/local_cache/",
+  "concurrency": 20,
+  "slack_webhook_url": "https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX",
+  "rabbitmq": {
+    "user": "guest",
+    "password": "guest",
+    "host": "rabbitmq",
+    "port": 5672,
+    "setup": {
+      "core": {
+        "commit_payload_exchange": "powerloom-backend-commit-payload:",
+        "exchange": "audit-protocol-backend"
+      },
+      "payload_commit": {
+        "queue_name_prefix": "powerloom-backend-commit-payload-queue:",
+        "routing_key_prefix": "powerloom-backend-commit-payload:"
+      }
+    }
+  },
+  "ipfs": {
+    "url": "/dns/ipfs/tcp/5001",
+    "reader_url": "/dns/ipfs/tcp/5001",
+    "write_rate_limit": {
+      "req_per_sec": 10,
+      "burst": 10
+    },
+    "timeout": 20,
+    "timeout_secs": 30,
+    "max_idle_conns": 5,
+    "idle_conn_timeout": 0
+  },
+  "redis": {
+    "host": "redis",
+    "port": 6379,
+    "db": 0,
+    "password": null
+  },
+  "redis_reader": {
+    "host": "redis",
+    "port": 6379,
+    "db": 0,
+    "password": null
+  },
+  "web3_storage": {
+    "url": "https://api.web3.storage",
+    "upload_url_suffix": "/upload",
+    "api_token": "web3-storage-token",
+    "timeout_secs": 30,
+    "max_idle_conns": 5,
+    "idle_conn_timeout": 0,
+    "rate_limit": {
+      "req_per_sec": 3,
+      "burst": 6
+    }
+  },
+  "signer": {
+    "domain": {
+      "name": "PowerloomProtocolContract",
+      "version": "0.1",
+      "chainId": "100",
+      "verifyingContract": "0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2"
+    },
+    "accountAddress": "0x05BE3003b0C5047DfEA4B7563Ce07B8fb268d721",
+    "privateKey": "",
+    "deadlineBuffer": 10
+  },
+  "relayer": {
+    "host": "0.0.0.0:8080",
+    "endpoint": "/endpoint"
+  },
+  "pruning": {
+    "ipfs_pinning_max_age_in_days": 7,
+    "local_disk_max_age_in_days": 7,
+    "cron_frequency": "0 0 */7 * *"
```

Comment:
Does it make sense to run the cron job once every 7 days? It is almost equivalent to accumulating 14 days worth of data instead of 7.

Reply:
We can run cron job every day, which will delete 7 (or configured value) days older snapshot data. Same goes for IPFS unpinning. WDYT?

Reply:
Changing default value to 1 day.